app-frontend/src/app/pages/editor/bucket/bucketedit.html

```diff
@@ -0,0 +1,92 @@
+<div class="container column-stretch">
+  <!-- Sidebar (container is in parent view) -->
+  <div class="sidebar scene-list-pane" ng-if="$ctrl.showLayerList" >
+    <!-- scene listing -->
+    <div class="sidebar-static">
+      <h5 class="sidebar-title">
+        Selected scenes: {{ $ctrl.selectedScenes.size }}
+      </h5>
+      <div class="sidebar-actions">
+        <select>
+          <option selected>Action</option>
+          <option><i class="icon-load"></i> Sync histogram</option>
+          <option><i class="icon-ccw"></i> Reset changes</option>
+          <option><i class="icon-trash"></i> Delete</option>
+        </select>
+        <button class="btn btn-square"
+                ng-class="{'btn-secondary': !$ctrl.shouldSelectAll()}"
+                ng-click="$ctrl.onToggleSelection()">
+          <i ng-class="{'icon-plus': $ctrl.shouldSelectAll(),
+                        'icon-cross': !$ctrl.shouldSelectAll()}"></i>
+        </button>
+      </div>
+    </div>
+    <div class="sidebar-scrollable" id="infscroll">
+      <div
+          infinite-scroll="$ctrl.getMoreScenes()" infinite-scroll-distance=".5"
+          infinite-scroll-disabled="$ctrl.loading || ($ctrl.lastSceneResult && !$ctrl.lastSceneResult.hasNext) || $ctrl.errorMsg"
+          infinite-scroll-container="'#infscroll'"
+          class="list-group">
+        <rf-scene-item
+            ng-click="$ctrl.openColorCorrect()"
+            scene="scene"
+            selectable
+            selected="$ctrl.isSelected(scene)"
+            on-select="$ctrl.setSelected(scene, selected)"
+            class="selectable"
+            ng-repeat="scene in $ctrl.sceneList track by scene.id">
+        </rf-scene-item>
+      </div>
+      <div class="list-group" ng-show="$ctrl.loading">
+        <span class="list-placeholder">
+          <i class="icon-load"></i>
+        </span>
+      </div>
+      <div class="list-group"
+           ng-show="!$ctrl.loading && $ctrl.lastSceneResult && !$ctrl.lastSceneResult.hasNext">
+        <span class="list-placeholder">
+          No more scenes to load
```

Comment:
Since this is most likely going to be scenes in a bucket, we don't need this message. We might want a placeholder if they don't have any scenes in the bucket, but we probably don't want to even allow them onto the page if the bucket contains no scenes.